python/ecl/grid/ecl_region.py

```diff
@@ -927,7 +928,7 @@ def idiv_kw( self , target_kw , other , force_active = False):
             else:
                 raise TypeError("Type mismatch")
         else:
-            self.scale_kw( target_kw , 1/other , force_active )
+            self.scale_kw( target_kw , ecl.python2_div(1, other) , force_active )
```

Comment:
This seem like it should really use a true division as it is a scaling function (floats make sense).

Reply:
I will try to use the suggested solutions here.

Reply:
The tests may be wrong if they rely on integer division. I had a similar issue when I first ported libecl to python3. I let the error be and used integer division, but made a comment in the code that the tests are wrong and created an issue on github.